### ISSUE TITLE
Improve exception message when passing int objects to method params

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1736]
Issue #1733 revealed that the TypeError that is raised when passing int objects is not easy to understand and needs to be improved:

```
Params = [
    ('ElementType', 2),
    ('InPool', sp_paths[0]),
    ('Size', 1024000),
    ('TheElement', None)
]
result, outparams = conn.InvokeMethod(
    'CreateOrModifyElementFromStoragePool',
    os_inst_path,
    Params)
```

This raises the following `TypeError`, due to the fact that the 'ElementType' parameter is specified as a Python `int` object in the input parameters of the method, when it should have been a `pywbem.Uint32` object:

```
Traceback (most recent call last):
File "", line 1, in
File "pywbem/cim_operations.py", line 4081, in InvokeMethod
result = self._methodcall(MethodName, ObjectName, Params, **params)
File "pywbem/cim_operations.py", line 1995, in _methodcall
ptuple = (p[0], p[1], infer_type(p[1]),
File "pywbem/cim_operations.py", line 1948, in infer_type
_format("Unsupported parameter type {0}", type(obj)))
TypeError: Unsupported parameter type <type 'int'>
```

The exception message does not really allow a user to understand what it even complains about, that they can correct it themselves, or how that would be done.

This should be improved.

**Comment by andy-maier** [comment]
Fixed by PR #1739.

**Comment by andy-maier** [comment]
PR #1743 rolls the fix back into 0.14.2.
